Assets/MixedRealityToolkit/UX/Scripts/AppBar/AppBarButton.cs

```diff
@@ -137,6 +137,7 @@ private void Hide()
             icon.Alpha = 0f;
             text.DisableText = true;
             cButton.enabled = false;
+            cButton.transform.Find("FrontPlate").GetComponent<MeshRenderer>().enabled = false;
```

Comment:
Let's cache this `FrontPlate`'s mesh renderer so we don't have to get the mesh renderer each time we Show/Hide